fsevents.js

```diff
@@ -11,11 +11,16 @@ if (process.platform !== 'darwin') {
 }
 
 const Native = require('./fsevents.node');
-const con = Native.constants;
+const events = Object.keys(Native.constants).reduce((obj, name) => {
+  obj[name.replace('kFSEventStreamEventFlag', '')] = Native.constants[name];
+  return obj;
+}, {});
 
 function watch(path, handler) {
-  if ('string' !== typeof path) throw new TypeError(`argument 1 must be a string and not a ${typeof path}`);
-  if ('function' !== typeof handler) throw new TypeError(`argument 2 must be a function and not a ${typeof handler}`);
+  if (typeof path !== 'string')
+    throw new TypeError(`fsevents argument 1 must be a string and not a ${typeof path}`);
```

Comment:
If the throw is not on the same line as the if should there be curly-braces around it for better readability or am I just being pedantic?

Reply:
Easy